azure_translator/errors.py

```diff
@@ -0,0 +1,41 @@
+"""Azure errors module."""
+import xml.etree.ElementTree as ET
+
+
+class BaseAzureException(Exception):
+    """
+    Base exception for the SDK.
+    """
+    def __init__(self, *args, **kwargs):
+        self.response = kwargs.pop('response', None)
+        self.request = kwargs.pop('request', None)
+        super(BaseAzureException, self).__init__(*args, **kwargs)
+
+
+class AzureApiError(BaseAzureException):
+    """
+    Raised when the API returns a non-200 body.
+    """
+    MSG_SEPARATOR = "; "
+
+    def __init__(self, msg, *args, **kwargs):
+        response = kwargs.get('response')
+        if response:
+            try:
+                msg = self.MSG_SEPARATOR.join(ET.fromstring(response.content).itertext())
```

Comment:
Could you include the status code with the extracted message? I'm comparing this with the raw `str(exception)`, and while the URL isn't too useful, the status code gives a bit of context.